airbrake/utils.py

```diff
@@ -155,7 +155,8 @@ def get_local_git_revision():
 def _git_revision_with_binary():
     """Get the latest git hash using the git binary."""
     try:
-        rev = subprocess.check_output(["git", "rev-parse", "HEAD"])
+        rev = subprocess.check_output(["git", "rev-parse", "HEAD"],
+                                      stderr=subprocess.STDOUT)
```

Comment:
Sorry, could would love to know the rationale here. 

So `fatal: Not a git repository (or any of the parent directories): .git` gets sent to stderr, and instead of allowing that to be printed, we set `rev` to this? Do we now need to check if the output is actually a revision value?